pkg/db/init.go

```diff
@@ -120,45 +119,14 @@ func Init() error {
 	}()
 
 	if RunMigrations {
-		if err := duty.Migrate(ConnectionString, nil); err != nil {
+		if err := duty.Migrate(ConnectionString, &migrate.MigrateOptions{IgnoreFiles: []string{"012_changelog.sql"}}); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
-func Cleanup() {
```

Comment:
Where does this happen now ?

Reply:
This func was never being called, creating a new PR for cleaning up old checks